NEWS.md

```diff
@@ -1,3 +1,7 @@
+# orderly 1.1.19
```

Comment:
should be 1.1.20 now. Consider rewording to reflect less the development process and more the user focus:

> Add support for running batches of orderly reports that differ in their parameters, available via `orderly::orderly_batch` and with the cli as `orderly batch` (VIMC-3603)

Reply:
Thanks, I've tidied that up